projects/igniteui-angular/src/lib/time-picker/time-picker.pipes.ts

```diff
@@ -0,0 +1,93 @@
+import { Pipe, PipeTransform, Inject} from '@angular/core';
+import { IGX_TIME_PICKER_COMPONENT, IgxTimePickerBase } from './time-picker.common';
+
+
+/** @hidden */
+@Pipe({ name: 'displayFormat'})
+export class TimeDisplayFormatPipe implements PipeTransform {
```

Comment:
Export those two pipes in the module so the users could use them. Add triple slash comments.